### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by dreschpe** [TheAntTeam/TheAntFarm#28]
One problem is the setting of the milling depth. If you use different PCB materials you have to run tests.
Solution - you can adjust the milling depth in real time with two buttons + - 0.01 mm.
You start with a low milling depth. The bit is only scratching the copper. You carefully increase the milling depth and look at the result.
If the milled isolation is ok you press stop and restart the process. The repeat accuracy of the ant is good. The new traces correct the scratches lines from the beginning.

Peter

**Comment by TheAntTeam** [comment]
Do you mean something like [Marlin Babystep](https://marlinfw.org/docs/gcode/M290.html)? (Implemented via software and not via firmware)

**Comment by IckyDeh** [comment]
I support this request as this would be super helpful for solder mask removal with spring loaded bit.

**Comment by TheAntTeam** [comment]
Yes, we also like this feature.  We just have to figure out how to insert it in a smart way.  Give us some time to think about it.
